pyodbc.cpython-36m-x86_64-linux-gnu.so,sha256=wD6-8TW6Brhgdp8q-ED8Bm4x6pu7mV1hDhIfnRW7SO0,684728
pyodbc-4.0.30.dist-info/DESCRIPTION.rst,sha256=-tzlb-whwFrbDan-AB3qzzq9YS3ScKnJdexN1DQ9swE,1181
pyodbc-4.0.30.dist-info/METADATA,sha256=FVQCYDqHY8nxLtbGJLgfTN1zLxud-5SNDf5TSnGMtZI,2123
pyodbc-4.0.30.dist-info/RECORD,,
pyodbc-4.0.30.dist-info/WHEEL,sha256=X8kVdBCq85ICewwfaE6btv5qKsFQfVq8NYJIXUK0i1A,104
pyodbc-4.0.30.dist-info/metadata.json,sha256=bl6pQH7xdZexojvTtsg2mZ_gPAzowxtt0GmY3hjRjcg,1020
pyodbc-4.0.30.dist-info/top_level.txt,sha256=30jvYcCul6_wA26OT5c068GAHzqQLpd9WfQ0mSSsDTU,7
pyodbc-4.0.30.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
